### PR TITLE
Add 2 presentations for 2 days of Oxford workshop

### PR DESCRIPTION
**Pull request by pwalczysko**
Presentations for 2 days of Oxford workshop.

cc @dominikl @jburel

### CODE REVIEW

File: 2019/Oxford/Day1/OMERO-workshop/index.html [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+                </section>
+
+
+                <section data-background-size="971px" data-background="images/omero-platform-bg-2.png">
+                    <h2>Metadata</h2>
+
+                    <div style="height: 350px">
+                    <ul>
+                        <li>Stored in a Relational Database</li>
+                        <li>Acquisition metadata</li>
```
Comment (mtbc) [inline 2019/Oxford/Day1/OMERO-workshop/index.html]: Maybe also capitalize "Metadata"?

**Comment by jburel** [comment]
Day 1: OMERO API slide no C++ in the list
Day 2: OMERO API slide C++ in the list
Day 1 is missing the slides about the data ownership setup

**Comment by pwalczysko** [comment]
@jburel This should fix the issues raised.

**Comment by jburel** [comment]
Thanks merging and deploying

**Pull request merged**